test/test_torch.py

```diff
@@ -3787,6 +3787,23 @@ def _test_cat_empty_legacy(self, use_cuda=False):
     def test_cat_empty_legacy(self):
         self._test_cat_empty_legacy(self)
 
+    def test_cat_padding(self):
+        x = torch.rand(13, 1, 3)
+        y = torch.rand(17, 2, 2)
+        z = torch.rand(19, 3, 1)
+
+        for pad_value in [0, 0.5, 1]:
+            res1 = torch.cat((x, y, z), 0, pad_value)
```

Comment:
torch.cat(tensors, pad_value=-1) is not working because you need to specify the dim. E.g. torch.cat(tensors, 0, pad_value=-1) works.

Reply:
emm, in the python API, dim has default value to 0, you don't need to specify it if you only want to set `pad_value` , it it possible to also make it default to 0 here if not specified? That is, to make `torch.cat(tensors, pad_value=-1)` work too. I think this would be more pythonic.

This is just my personal suggestion. I'm not a team member, so you can choose to take or ignore my suggestion.